mne/annotations.py

```diff
@@ -88,3 +88,19 @@ def _combine_annotations(annotations, last_samps, first_samps, sfreq):
     duration = np.concatenate([old_duration, annotations[1].duration])
     description = np.concatenate([old_description, annotations[1].description])
     return Annotations(onset, duration, description, old_orig_time)
+
+
+def _onset_to_seconds(raw, onset):
+    """Helper function for adjusting annotation onsets in relation to raw data.
+    """
```

Comment:
could you document the parameters too? Even with private functions, it's convenient to have full documentation.